src/symbols/Edge.tsx

```diff
@@ -68,14 +68,15 @@ export const Edge: FC<EdgeProps> = ({
   const { isSelected, hasSelections, hasSingleSelection } = useStore(state => ({
     hasSingleSelection: state.selections?.length === 1,
     hasSelections: state.selections?.length,
-    isSelected: state.selections?.includes(fromId) || state.selections?.includes(id)
+    isSelected:
+      state.selections?.includes(fromId) || state.selections?.includes(id)
   }));
 
   const selectionOpacity = hasSelections
     ? isSelected && hasSingleSelection
-      ? 1
-      : 0.1
-    : 1;
+      ? theme.edge.opacityOnlySelected || 1
```

Comment:
If the value is `0` this won't work. I think its safe to remove these defaults given its not likely this would be undefined.

Reply:
defaults removed